src/main/webapp/app/shared/image/session-storage-strategy.ts

```diff
@@ -5,7 +5,7 @@ import { GlobalCacheConfig, ICachePair, IStorageStrategy } from 'ts-cacheable';
  * instead of the localStorage: https://github.com/angelnikolov/ts-cacheable/blob/master/common/LocalStorageStrategy.ts
  */
 export class SessionStorageStrategy extends IStorageStrategy {
-    private masterCacheKey: string = GlobalCacheConfig.globalCacheKey;
+    private mainCacheKey: string = GlobalCacheConfig.globalCacheKey;
```

Comment:
I don't think the changes in this file were intentional.

Reply:
They actually were intentional. The term master should be avoided completely if possible